tests/unit/test_config.py

```diff
@@ -1131,7 +1142,11 @@ def test_apply_cloud_providers_config_extend_multiple(self):
                           {'password': 'new-password',
                            'extends': 'my-dev-envs:linode',
                            'location': 'Salt Lake City'
-                          }],
+                          },
+                          {'extends': 'my-dev-envs:tencentcloud',
+                           'id': 'new-id',
+                           'key': 'new-key',
+                           'location': 'ap-beijing'}],
```

Comment:
if you use 
```suggestion
                           'location': 'ap-beijing'},
        ],
```
(not sure about indent levels) the next change won't have a silly blame - like `),` that you'll have on line 1145 :trollface: